src/models/User.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+const users = [];
+
+const get = () => {
+  return users;
+};
+
+const getById = ({ id }) => {
+  return users.find(user => user.id === id) || null;
+};
+
+const create = ({ name }) => {
+  let nextID = 0;
+
+  if (users.length > 0) {
+    nextID = users.sort((a, b) => a.id - b.id).at(-1).id + 1;
+  }
+
+  users.push({
+    id: nextID, name,
+  });
+
+  return getById({ id: nextID });
+};
+
+const update = ({ id, name }) => {
+  const user = getById({ id });
```

Comment:
It should check in controller in this case if the user exists or not if not return an error otherwise do update function